lib/videoroom_web/templates/stats/index.html.heex

```diff
@@ -0,0 +1,108 @@
+<ul id="root">
+</ul>
+
+<style>
+ul {
+  list-style: inside;
+  padding-left: 30px
+}
+
+canvas {
+  width: 200px;
+  height: 200px
+}
+</style>
+
+<script src="https://cdn.jsdelivr.net/npm/phoenix@1.5.4/priv/static/phoenix.min.js"></script>
```

Comment:
It's still different :stuck_out_tongue: 
https://github.com/membraneframework/membrane_videoroom/blob/61efdc378814065be9b41282d9ffdc7f3393aadc/mix.lock#L74